packages/components/src/ellipsis-menu/menu-item.js

```diff
@@ -20,26 +20,28 @@ class MenuItem extends Component {
 
 	onClick( event ) {
 		const { isClickable, onInvoke } = this.props;
-		if ( isClickable ) {
+		if ( isClickable && onInvoke ) {
 			event.preventDefault();
 			onInvoke();
 		}
 	}
 
 	onKeyDown( event ) {
-		if ( event.keyCode === ENTER || event.keyCode === SPACE ) {
+		const { onInvoke } = this.props;
+
+		if ( onInvoke && ( event.keyCode === ENTER || event.keyCode === SPACE ) ) {
```

Comment:
Oddly, this isn't working on `ENTER` for me either.

Reply:
Looks like `onInvoke` is undefined so this condition isn't passing.